src/manipulation.js

```diff
@@ -199,7 +199,9 @@ function domManip( collection, args, callback, ignored ) {
 
 							// Optional AJAX dependency, but won't run scripts if not present
 							if ( jQuery._evalUrl && !node.noModule ) {
-								jQuery._evalUrl( node.src );
+								jQuery._evalUrl( node.src, {
+									nonce: node.nonce || node.getAttribute( "nonce" )
```

Comment:
I hoped this would be compressable with the other similar logic in `DOMEval.js` but there we iterate over more properties so it cannot be generalized so easily.

We'll be able to simplify it when Firefox & Safari implement the `nonce` property.

Reply:
cc @gibson042 for compressability efforts